src/components/attendee/Button/Button.tsx

```diff
@@ -21,6 +25,7 @@ export const Button: React.FC<ButtonProps> = ({
     [CN.buttonPrimary]: variant === "primary",
     [CN.buttonAlternative]: variant === "alternative",
     [CN.buttonAlternativeBorder]: variant === "alternativeBorder",
+    [CN.buttonPrimaryAlternate]: variant === "primaryAlternate",
```

Comment:
These don't look like general variants like `primary`, `secondary`...  

If there are combinations allowed, might be best having two or three different props to `Button`.

Not just `variant`, but maybe `effect` with values `normal`, `glass`, a prop named `border` with values `none`, `slim` and maybe then it will not have to be hard-coded to only white border.

Reply:
button props/variant redone [a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)